README.md

```diff
@@ -121,6 +122,9 @@ Certain objects were changed after they were added to `typing`, and
   introspectable at runtime. In order to access overloads with
   `typing_extensions.get_overloads()`, you must use
   `@typing_extensions.overload`.
+- `NamedTuple` was changed in Python 3.11 to allow for multiple inheritance
+  with `typing.Generic` The implementation of `NamedTuple` was also changed in
+  3.9 so that `NamedTuple` became a function rather than a class.
```

Comment:
Maybe switch it around so that the earlier change is earlier in the text?

```suggestion
- The implementation of `NamedTuple` was changed in Python 3.9 so that `NamedTuple` became
  a function rather than a class. Since Python 3.11 `NamedTuple` allows for multiple inheritance
  with `typing.Generic` 
```

(Although the 3.11 change is of course the more relevant one.)

Reply:
Yeah, the 3.9 change is basically an implementation detail that's only being backported to keep the `typing_extensions` code simple (I didn't want to have to introduce a whole new code branch so that `typing_extensions.NamedTuple` would match `typing.NamedTuple` and be a class in 3.8). I'd rather lead with the 3.11 change, I think.